### DISCUSSION TITLE
基于Ubuntu打造的Windows12系统！【整活】

### DISCUSSION THREAD

**Discussion by User782Tec** [win12-online/win12#323] [Ideas]
![Screenshot_2023-11-19-21-55-45-314_com miui video](https://github.com/tjy-gitnub/win12/assets/115236204/43000d70-c8c6-42b6-95a4-271db3799e36)

技术实现@NB-Group

**Comment by ghost** [comment]
镜像呢？我下个

**Comment by ghost** [comment]
镜像呢？我下个

**Comment by ghost** [comment]
镜像呢？我下个

**Reply by NB-Group** [reply]
镜像没有，20个G的虚拟机文件倒是有，就看你有没有耐心等它下载完了

**Reply by ghost** [reply]
链接呢

**Comment by ghost** [comment]
没事,我C 盘还有973个G
Vmware主板要BIOS or UEFI?

**Reply by ghost** [reply]
（我的主板是UEFI，不过我可以解压）

**Comment by ghost** [comment]
最近做了一个带着虚拟机的Win 10 PE可以运行了😁😁😁

**Comment by ghost** [comment]
镜像约一个G

**Comment by Mbilse** [comment]
> 没事,我C 盘还有973个G
> Vmware主板要BIOS or UEFI?

(！)
《C:/   可用:973GB》

**Reply by LGOS-Studio** [reply]
这是我上一个账号，说明：这是我WTG盘有973GB/999GB

**Reply by kjmjh** [reply]
我的新力作:https://github.com/kjmjh/skyaitrainer

**Reply by Mbilse** [reply]
> 我的新力作:https://github.com/kjmjh/skyaitrainer

我的Mbilse Care接入Mbilse Cloud了
https://mbilse001.github.io

**Reply by kjmjh** [reply]
啊这…你要搞云服务?

**Reply by tangyuan0821** [reply]
> > 我的新力作:https://github.com/kjmjh/skyaitrainer
> 
> 我的Mbilse Care接入Mbilse Cloud了
> https://mbilse001.github.io
> 

6

**Reply by Mbilse** [reply]
> 啊这…你要搞云服务?

yes

**Reply by Mbilse** [reply]
> 啊这…你要搞云服务?

甚至还做了一个 电脑教室 - 课堂爆破者

https://github.com/Mbilse/computer-room-class-breaker

**Reply by tangyuan0821** [reply]
祝你好运！（不过我们离题了）

**Reply by kjmjh** [reply]
…

**Reply by lingbopro** [reply]
一排 off topic 好喜感（

**Reply by tangyuan0821** [reply]
> 一排 off topic 好喜感（

哈哈
